tools/syncer.md

```diff
@@ -73,25 +73,44 @@ batch = 10
 # The testing address for pprof. It can also be used by Prometheus to pull the syncer metrics.
 status-addr = ":10081"
 
-skip-sqls = ["ALTER USER", "CREATE USER"]
+# Notice: skip-sqls is abandoned, please use skip-ddls instead.
+# skip-ddls skip ddl statements which supports regular expression. When there are DDL sqls incompatible with tidb, you should skip them via these rules.
+# skip-ddls = ["^CREATE\\s+USER"]
+
+# Notice: skip-events is abandoned, please use skip-dmls instead
+# skip-dmls skip DML statements. type value can be 'insert', 'update', 'delete'.
```

Comment:
-> skip-dmls skips DML statements. The type value can be 'insert', 'update' and 'delete'.